content/darkroom/masking-and-blending/masks/raster.md

```diff
@@ -7,17 +7,19 @@ draft: false
 
 As described in the previous sections, the final output of a module's mask (the combined effect of any drawn and parametric masks) is a grayscale raster image representing the extent to which the module's effect should be applied to each pixel. This raster image is stored internally for active modules and can be subsequently reused by other modules in the pixelpipe.
 
-As with any mask, if the opacity value for a pixel in a raster mask is zero the module's input passed through the module unchanged. If the opacity is 1.0 the module has its full effect. For each value between 0 and 1.0 the module's effect is applied proportionally at that location.
+As with any mask, if the opacity value for a pixel in a raster mask is zero, the module's input is passed through the module unchanged. If the opacity is 1.0 the module has its full effect. For each value between 0 and 1.0 the module's effect is applied proportionally at that location.
 
-You can choose a raster mask from the combobox. Raster masks can be identified by the name of the module against which they were originally generated. 
+You can choose a raster mask from the combobox. Raster masks can be identified by the name of the module against which they were originally generated.
 
 ---
 
-**Note:** Raster masks are generated as part of a module's internal processing. Once a module's processing is complete its mask then becomes available to subsequent modules in the pixelpipe. 
+**Note 1:** See [external-raster-mask](../module-reference/processing-modules/external_raster.md) module.
+
+**Note 2:** Raster masks are generated as part of a module's internal processing. Once a module's processing is complete it's mask then becomes available to subsequent modules in the pixelpipe.
 
 This has two implications:
 
-1. Raster masks cannot be generated by disabled modules since they do not participate in pixelpipe processing. As soon as you disable a module, its mask is no longer available for use.
+1. Raster masks cannot be generated by disabled modules since they do not participate in pixelpipe processing. As soon as you disable a module, it's mask is no longer available for use.
```

Comment:
The original was correct: "its" is possessive; "it's" is short for "it is"

Reply:
Isn't this a genitive here?

Reply:
I think "it" is like the exception to the rule here, so if you want to rewrite it as "the module's mask is no longer available" you should use an apostrophe but the possessive of it is "its"